tests/scanners/test.html.js

```diff
@@ -57,6 +57,41 @@ describe('HTML', function() {
       });
   });
 
+  it('should warn on remote <script> tag src attribute', () => {
+    var badHTML = validHTML(singleLineString`
+      <script src="http://foo.bar/my.js"></script>
+      <script src="https://foo.bar/my.js"></script>
+      <script src="file://foo.bar/my.js"></script>
+      <script src="ftp://foo.bar/my.js"></script>
+      <script src="moz-extension://foo/my.js"></script>
```

Comment:
Can we please add the following test as well?

`<script src="//foo.bar/my.js"></script>`